sympy/printing/tests/test_octave.py

```diff
@@ -354,6 +354,12 @@ def test_sparse():
     )
 
 
+def test_sinc():
+    assert mcode(sinc(x)) == 'sinc(x/pi)'
+    assert mcode(sinc((x + 3))) == 'sinc((x + 3)/pi)'
+    assert mcode(sinc(pi*(x + 3))) == 'sinc((x + 3))'
```

Comment:
Here the correct output is `'sinc(x + 3)'` not `'sinc((x + 3))'`.

You can locally test files before committing. Just run `bin/test <FILENAME>`
e.g., `bin/test octave`.